stb_vorbis.c

```diff
@@ -1434,6 +1457,19 @@ static int set_file_offset(stb_vorbis *f, unsigned int loc)
    #endif
 }
 
+static __forceinline uint64 get64raw(uint8 *f)
+{
+   uint64 x;
+   x = f[0];
+   x += f[1] << 8;
```

Comment:
You need a cast here as well. uint8 shifted left by 8 is 0.

Reply:
It is false that you need casts here, uint8 shifts are promoted to int before shifting, so shift by 8 is not 0. (Technically shift by 24 does though, since it promotes to int and then you're shifting into the sign bit, which is why get32() already had a cast for the shift-by-24 step.)

Reply:
Ah, thanks for the info.